<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotDialog</class>
 <widget class="QDialog" name="ScreenshotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>761</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>761</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Screenshot Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../nexus.qrc">
    <normaloff>:/images/favicon.png</normaloff>:/images/favicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#ScreenshotDialog {
    background: #15181f
}
QLabel {
	color:#ddd
}
* {
	outline: none;
	font:11px Arial;
}

QProgressBar {
    border: none;
    border-radius: 2px;
	text-align:center;
    background: #323947;
	color:#fff;
}

QProgressBar::chunk {
    background-color: #24dba1
}

QPushButton {
    background: #2574ab;
    border-radius: 2px;
	padding: 3px 5px 2px;
    border: none;
    min-width: 100px;
    color: #ffffff;
}

QPushButton::hover {
    background: #2b85c4;
}

QPushButton::pressed {
    background: #206291;
}

QPushButton:disabled {
    background-color: #1c2029;
    color: #46474a;
}

QListView {
	padding:4px;
	padding-left:2px;
    background: #15181f;
	border-radius:2px;
	border: 1px solid #323947;
    show-decoration-selected: 1; /* make the selection span the entire width of the view */
}

QListView::item {
    border: 0px;
	padding:-3px;
	padding-bottom:-4px;
	padding-right:-4px;
	margin:3px;
	margin-bottom: 4px;
}

QListView::item:selected:!active {
    background: #24dba1;
}

QListView::item:selected:active {
    background: #24dba1;
}

QListView::item:hover {
    background: #24dba1;
}


QScrollBar {
    border: none;
    background: #20252e;
	border-radius:2px;
    width: 5px;
    margin: 0;
}

QScrollBar::handle {
    background: rgba(255,255,255,0.1);
    min-height: 20px;
	border-radius:2px;
}

QScrollBar::handle:hover {
    background: rgba(36,219,161,1);
    min-height: 20px;
	border-radius:2px;
}

QScrollBar::sub-line, QScrollBar::add-line{
    subcontrol-origin: margin;
    height: 20px;
    border: none;
    background: transparent;
	border-radius:2px;
}

QScrollBar::up-arrow, QScrollBar::down-arrowl {
    border: none;
}

QScrollBar::add-page, QScrollBar::sub-page {
    background: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">border: 1px solid #323947;
border-radius:2px;
padding:10px 12px 9px;
color:#ccc</string>
     </property>
     <property name="text">
      <string>Uploading screenshots can take a while depending on their size and how many you upload at a time. To keep things quick, limit yourself. &lt;a href=&quot;http://sto.gamepedia.com/Screen_shot&quot; style=&quot;color:#24dba1&quot;&gt;Learn More&lt;/a&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignTop">
    <widget class="QFrame" name="frame_3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>258</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>258</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>96</width>
             <height>96</height>
            </size>
           </property>
           <property name="movement">
            <enum>QListView::Static</enum>
           </property>
           <property name="flow">
            <enum>QListView::LeftToRight</enum>
           </property>
           <property name="isWrapping" stdset="0">
            <bool>true</bool>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="stats">
           <property name="styleSheet">
            <string notr="true">
padding:5px;</string>
           </property>
           <property name="text">
            <string>Screenshots Selected: &lt;strong&gt;%0 / %1&lt;/strong&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="frame_2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignTop">
          <widget class="QLabel" name="preview">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>258</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>258</height>
            </size>
           </property>
           <property name="text">
            <string>Preview</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="caption">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#15181f;
border:1px solid #323947;
color: #fff;
padding:5px;</string>
           </property>
           <property name="placeholderText">
            <string>Add a caption to your screenshot (Optional)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>50</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Processing, please be patient...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>{Spinner Placeholder}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QFrame" name="frame_4">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="select_all">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Select All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openFolder">
           <property name="text">
            <string>Open Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="onlineGallery">
           <property name="text">
            <string>Online Gallery</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_9">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="delete_selected">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="upload_selected">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Upload</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../nexus.qrc"/>
 </resources>
 <connections/>
</ui>
